tests/config/test_agent_config.py

```diff
@@ -35,24 +35,41 @@ def test_agent_config_initialization():
     assert config.force_env is None
     assert config.env.window.headless is False
 
+def test_gemini_method():
```

Comment:
can you also add a test when it fails ? With an assert_raises ? otherwise LGTM. Thanks for the help !